README.md

```diff
@@ -104,6 +104,53 @@ In case of any architecture violations, the actual architecture can be visualize
 $ dot -Tpng kroxylicious/target/generated-sources/annotations/deptective.dot > kroxylicious-arch.png
 ```
 
+
+# Setting Up in Windows Using WSL
+### For installing WSL (Skip this step if already done)
+1. Enable the Windows Subsystem for Linux feature: To enable WSL, you need to enable the Windows Subsystem for Linux feature in the Windows Features dialog.
+
+
+2. Install a Linux distribution from the Microsoft Store(Ubuntu preferably)
```

Comment:
Is there a particular reason to prefer Ubuntu?

Reply:
User-friendliness, Frequent Releases: Ubuntu is released every six months, while Debian releases a new version every two to three years and Community: Ubuntu has a large and active community

Reply:
I'd say Fedora meets those criteria and none of them are related to running or building Kroxylicious. So I don't think its appropriate for Kroxylicious to recommend a distribution.